IM_calculation/scripts/calculate_ims.py

```diff
@@ -163,6 +163,25 @@ def load_args():
     )
 
     args = parser.parse_args()
+
+    if constants.Components.ceas.str_value in args.components:
+        wrong_ims = ""
+        if "FAS" not in args.im:
+            wrong_ims = ",".join(args.im)
+        else:
+            # there are IMs other than FAS but user only wants EAS
+            # without this check, the code still runs, but the non-FAS IMs
+            # won't be shown in the output csv without a notice/warning
+            if len(args.im) > 1 and len(args.components) == 1:
+                args.im.remove("FAS")
+                wrong_ims = ",".join(args.im)
+        if wrong_ims:
+            parser.error(
+                "The specified IMs need non-EAS components to proceed: {}".format(
```

Comment:
We should say that EAS needs FAS if we're going to go down this route.
However I think this should be handled more like components.
If FAS is given we calculate and save FAS.
If ESA is given we calculate FAS, use it to compute EAS and save only that.
If both are given we calculate and save both.

Reply:
EAS is a separate component, only used by FAS.
>If FAS is given we calculate and save FAS: Yes for the specified component. 
>If ESA is given we calculate FAS, use it to compute EAS and save only that: No, EAS is not a stand-alone. If EAS is specified as a desired component but FAS is not in the IM list, an error is thrown.
>If both are given we calculate and save both: Yes, It will compute EAS component of FAS IM type.